corehq/apps/hqadmin/service_checks.py

```diff
@@ -100,11 +100,15 @@ def check_elasticsearch():
            'date': datetime.datetime.now().isoformat()}
     send_to_elasticsearch('groups', doc)
     refresh_elasticsearch_index('groups')
-    hits = GroupES().remove_default_filters().doc_id(doc['_id']).run().hits
+    bad_hosts = []
+    for host_config in es_hosts():
+        hits = GroupES(debug_host=host_config).remove_default_filters().doc_id(doc['_id']).run().hits
```

Comment:
I think this will only work if group has a shard on each node which could cause false positives

Reply:
hmm, good point